.github/ISSUE_TEMPLATE/helm_feature.md

```diff
@@ -0,0 +1,75 @@
+name: Loki Helm Chart Feature Request
+description: Suggest an idea for the Loki Helm chart
+title: "[loki] Helm Feature Request Title"
+labels: [enhancement, area/helm]
+assignees:
+  - 
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request for the Loki Helm chart!
+        
+        For general Loki usage questions, please use the [Grafana Community Slack](https://slack.grafana.com/) and join the `#loki` channel. This issue tracker is for feature requests specific to the Loki Helm chart.
+
+  - type: input
+    id: chart-name
+    attributes:
+      label: Which chart?
+      description: Enter the name of the chart for this feature request (e.g., loki, loki-stack, loki-distributed). Note we only `officially` support the `loki` chart.
+      placeholder: loki
+    validations:
+      required: true
+
+  - type: dropdown
+    id: deployment-mode
+    attributes:
+      label: What's your Loki deployment mode?
+      description: Select the deployment mode this feature request relates to
+      options:
+        - Single Binary (monolithic)
```

Comment:
I like that this includes different ways of describing the deployments.  That's helpful!